ambari-common/src/main/python/resource_management/libraries/script/script.py

```diff
@@ -210,6 +210,25 @@ def get_config_dir_during_stack_upgrade(self, env, base_dir, conf_select_name):
       return os.path.realpath(config_path)
     return None
 
+
+  def save_mpack_to_structured_out(self, command_name):
+    """
+    Writes out information about the managment pack which was installed if this is an installation command.
+    :param command_name: command name
+    :return: None
+    """
+    is_install_command = command_name is not None and command_name.lower() == "install"
+    if not is_install_command:
+      return
+
+    command_repository = CommandRepository(self.get_config()['repositoryFile'])
+
+    Logger.info("Reporting installation state for {0}".format(command_repository))
+
+    self.put_structured_out({"mpackId": command_repository.mpack_id})
+    self.put_structured_out({"mpackName":command_repository.mpack_id})
```

Comment:
Is this for the future?  The id is the mpackName?

Reply:
No, this is a bug - nice catch. Turns out we're not actually using the mpack name in the structured output, but I'll update it to be correct.